components/webgpu/wgpu_thread.rs

```diff
@@ -694,10 +694,6 @@ impl WGPU {
                             })
                             .map_err(|e| e.to_string());
 
-                        if response.is_err() {
-                            break;
-                        }
-
```

Comment:
Would it make sense to match this error response and `warn!()` as we do when the send immediately following it fails?

Reply:
No, because we send the error (or Ok(...)) to script (dom), where error get's handled (promise reject), while IPC errors does not get handled at all (apart from printing warning).